index.html

```diff
@@ -1,14 +1,60 @@
-<!doctype html>
-<head>
-<title>JSProjects by Neha</title>
-</head>
-<body>
-<section class="container">
-  
-  <h2>Javascript Projects</h2>
-  <ul>
-    <li><a href="js-tipcalculator/index.html" >Tip Calculator</a></li>
-    <li>Cost spliter calculator</li>
-  </ul>
-</section></body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>JSProjects by Neha</title>
+        <style>
```

Comment:
To make it more prettier:

- We should create a _resource_ folder to have _css_ and _JS_ into that.

Thanks